pygdf/dataframe.py

```diff
@@ -27,55 +27,76 @@ class DataFrame(object):
     Examples
     --------
 
-    Build dataframe with `__setitem__`
-
-    >>> from pygdf.dataframe import DataFrame
-    >>> df = DataFrame()
-    >>> df['key'] = [0, 1, 2, 3, 4]
-    >>> df['val'] = [float(i + 10) for i in range(5)]  # insert column
-    >>> df
-      key val
-    0 0   10.0
-    1 1   11.0
-    2 2   12.0
-    3 3   13.0
-    4 4   14.0
-    >>> len(df)
-    5
-
-    Build dataframe with initializer
-
-    >>> import numpy as np
-    >>> df2 = DataFrame([('a', np.arange(10)),
-    ...                  ('b', np.random.random(10))])
-    >>> df2
+    Build dataframe with `__setitem__`:
+
+    .. code-block:: python
+
+        from pygdf.dataframe import DataFrame
+        df = DataFrame()
+        df['key'] = [0, 1, 2, 3, 4]
+        df['val'] = [float(i + 10) for i in range(5)]  # insert column
+        df
+
+    Output:
+
+    .. code-block:: python
+
+          key val
+        0 0   10.0
+        1 1   11.0
+        2 2   12.0
+        3 3   13.0
+        4 4   14.0
+
+    Build dataframe with initializer:
+
+    .. code-block:: python
+
+      from pygdf.dataframe import DataFrame
+      import numpy as np
+      df = DataFrame([
+        ('a', np.arange(5)),
+        ('b', np.random.random(5))
+      ])
+      df
+
+    Output:
+
+    .. code-block:: python
+
+        a b
+      0 0 0.777831724018
+      1 1 0.604480034669
+      2 2 0.664111858618
+      3 3 0.887777513028
+      4 4 0.55838311246
+
+    Convert from a Pandas DataFrame:
+
+    .. code-block:: python
+
+      import pandas as pd
+      from pygdf.dataframe import DataFrame
+      pdf = pd.DataFrame({
+        'a': [0, 1, 2, 3],
+        'b': [0.1, 0.2, None, 0.3]
+      })
+
+      >>> pdf
```

Comment:
I don't think it makes any sense to mix blocks like this, especially considering that `python` and `pycon` (Python Console) are different pygments lexers: http://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer